fat32_cp.sh

```diff
@@ -8,7 +8,7 @@ whoami
 if [ "$(whoami)" = "root" ]; then
 	echo "Am root"
 	mount -o utf8=true $IMAGE_PATH_DIR test/fat32/mnt/
-	cp -r test/fat32/test_files/. test/fat32/mnt/
+	cp -r test/fat32/test_files/. test/fat32/mnt/ &&
```

Comment:
What does this do?

Reply:
Yeah I've had that issue before as well. We're hoping to move away from mounting and use mtools.